src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -340,6 +340,7 @@ public String toString() {
     SEND_TIME_AS_DATETIME("sendTimeAsDatetime", true),
     TRANSPARENT_NETWORK_IP_RESOLUTION("TransparentNetworkIPResolution", true),
     TRUST_SERVER_CERTIFICATE("trustServerCertificate", false),
+    USE_PLATFORM_GSS_CREDENTIALS("usePlatformGssCredentials", false),
```

Comment:
Let's also add this new connection property to SQLServerDataSource setter/getter API (also to interface) with Documentation in Javadocs